climada/util/calibrate/base.py

```diff
@@ -158,6 +159,81 @@ def plot_impf_set(self, **plot_kwargs):
         """
         return self.impf_set.plot(**plot_kwargs)
 
+    def plot_impf_variability(
+        self,
+        cost_func_diff: float = 0.1,
+        p_space_df: Optional[pd.DataFrame] = None,
+        plot_haz: bool = True,
+        **plot_kwargs
```

Comment:
`**plot_kwargs` are only passed to the first, but not the second plot. Maybe we should add to arguments, like `plot_impf_kws` and `plot_hist_kws`

Reply:
Sounds good. For now, I didn't add a third kwargs argument for the impact functions in grey, so that's the only thing that still cannot be changed, but it may be an overkill to add this too. 
With plot_impf_kws and plot_hist_kws the plot can be adjusted quite flexibly.